README.md

```diff
@@ -1,47 +1,103 @@
+
+## Elementor Roadmap
+
+Discover what's Next in Elementor. Visit the [Elementor Roadmap](https://elemn.to/gh-to-roadmap) and learn about the status of upcoming or newly-released features.
 
 ## Developer API
-Elementor website builder is free and open source. It’s the perfect plugin to be extended and integrated further. It was created and developed with the developer in mind, and we have already seen some truly remarkable addons being developed.
 
-To help you get started and learn just how to integrate with Elementor, we have created the official [Elementor Developers Center](https://developers.elementor.com/?utm_source=github-repo&utm_medium=link&utm_campaign=readme). Check the [Developers Blog](https://developers.elementor.com/blog/?utm_source=github-repo&utm_medium=link&utm_campaign=readme) to discover the latest information from the programmers working on the newest Elementor versions, or visit the [Developers Docs](https://developers.elementor.com/docs/?utm_source=github-repo&utm_medium=link&utm_campaign=readme) to learn how to extend Elementor core and create your own addons.
+Elementor website builder is free and open source. It's the perfect plugin to extend and integrate further. It was envisioned with the developer in mind, and we have already seen some truly remarkable addons created by many skilled developers.
+
+To help you get started and learn just how to integrate with Elementor, we have created the official [Elementor Developers Center](https://elemn.to/gh-to-dev-center).
+
+Check the [Developers Blog](https://elemn.to/gh-to-dev-blog) to discover the latest information from our engineers working on the newest Elementor versions, or visit the [Developers Documentation](https://elemn.to/gh-to-dev-docs) to learn how to extend Elementor and create your Addons.
 
-## Be a contributor
-Community contributions are always welcome and help us remain the Number 1 WordPress Website Builder, feel free to answering questions on GitHub and within the various Elementor communities.
+#### Developers Community
+
+[Join our Developers Community](https://elemn.to/dev-community) and share tips, tricks, and knowledge with other developers about extending and developing Addons for Elementor.
+
+## Be a Contributor
+
+Community contributions are always welcomed and help us remain the Number 1 WordPress Website Builder. Feel free to answer questions on GitHub and within the various Elementor communities.
 
 There are many ways to contribute to Elementor:
 
-**Bug Report**: If you find a bug in the source code, you can help us by [submitting a bug report](https://github.com/elementor/elementor/issues/new?template=Bug_report.md). Even better, you can [submit a Pull Request](https://github.com/elementor/elementor/blob/master/.github/CONTRIBUTING.md#before-submitting-a-pr) with a fix.
+#### Contribution Guidelines
```

Comment:
You are jumping from H2 to H4?

Reply:
Oops, you're right. I missed that one! I'm going to fix this.